client/state/preferences/README.md

```diff
@@ -45,5 +45,6 @@ To add a new preference key, you only need to add a key and defaut value in `DEF
 ### Actions
 
 - **fetchPreferences()** - populate the tree
-- **setPreference( key, value )** - set preference and persist all preferences in the endpoint
+- **savePreference( key, value )** - set preference and persist all preferences in the endpoint
```

Comment:
I wonder if we should just key off of the preference constants to see if an api request should be sent out, so we could use the setPreference action for all preferences. Would we ever want to sometimes store a preference for a session, and later store it forever?